thirdparty/faiss/faiss/impl/index_read.cpp

```diff
@@ -512,7 +594,8 @@ static void read_HNSW(HNSW* hnsw, IOReader* f) {
     READVECTOR(hnsw->cum_nneighbor_per_level);
     READVECTOR(hnsw->levels);
```

Comment:
Don't these require mmap? Although these only account for a small part

Reply:
same question, what do we exactly mmap for hnsw @alexanderguzhva